README.md

```diff
@@ -41,7 +46,7 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- Replace this comment with your answer -->
+a) The variables do not change, only their values which both are 10 by the end of the code.  Initially x started with a value of 4 and y with a value of three.  The third line assigned a value of 3 (since x now equaled y, which had a value of 3) and then, y was changed to 10, and since x=y, x and y were assigned a value of 10.
```

Comment:
Not quite. `x = y` does not form any sort of persistent relationship between `x` and `y`; instead, `y` is evaluated first, and the value of `y` gets assigned to `x`.